### PR TITLE
Generate test environment

### LINKED ISSUE #117: add script to generate test environment
Script inputs:
- user already logged in to Azure subscription
- resource group name (preferably created)
- variables: storage account name, keyvault name, identity name

Script output:
- create all required resources (SA + containers, KV x2, MI)
- setup all required accesses
- setup all required items inside resources (blobs, secrets, etc.)

Other ideas:
- all parameters can be optional, so it will generate some default names like: azmitestKV, azmitestRG, ...
- some parameters need to have number or random chars, due to unique DNS requirements (KV, SA), so it will be like azmitestSA-00233 (increasing number of characters)

During this task we can also simplify naming of tested / created resources.

### PR DESCRIPTION
**Pull request by iricigor**
List of scripts that enable to generate test environments.
Check out basic usage in [README](https://github.com/SRE-PRG/azmitool/tree/gen-test-env/test/environment).

Testing will require to check-out branch and try to generate environment as described in basic usage (two PowerShell commands)

I need to merge before finalizing it in order to pick up new tests and setup accordingly new environment.

Also, we need to discuss for final version on naming certain things in our tests:
- remove neu-pre and similar stuff, or 
- simplify names or values of certain other things
- splitting integration testing into two parts: functional tests and package tests

Resolves #117

### CODE REVIEW

File: test/environment/AzmiEnvironment.psd1 [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+ModuleVersion = '0.1'
+
+# Supported PSEditions
+CompatiblePSEditions = @('Desktop', 'Core')
+
+# ID used to uniquely identify this module
+GUID = 'b89e14bb-6cfb-4da3-8ce7-35d2b0ad6ffe'
+
+# Author of this module
+Author = 'iricigor@gmail.com'
```
Comment (MangyCoyote) [inline test/environment/AzmiEnvironment.psd1:24]: Do you want this?

File: test/environment/README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+## Contents
+
+- Basic Usage
+- Required permissions
+- Commands
+- Test VMs
+
+## Basic Usage
+
+Prior to using this PowerShell module you need to import it.
```
Comment (MangyCoyote) [inline test/environment/README.md:15]: Going step by step and will comments firs-go's perspective.
- Do I need admin rights? Usually sw requires it, so here could be good to tell user "no admin rights" required in a commentary.

File: test/environment/README.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+```PowerShell
+git clone https://github.com/SRE-PRG/azmitool # skip if you already have it clonned
+
```
Comment (MangyCoyote) [inline test/environment/README.md:19]: missing `cd azmitool`

File: test/environment/README.md (hunk 2 above)
Comment (iricigor) [reply test/environment/README.md]: Updated

File: test/environment/README.md (hunk 3 above)
Comment (iricigor) [reply test/environment/README.md]: Thanks for spotting it! I updated `Import-Module` command, which is enough.

**Pull request merged**